falco/values.yaml

```diff
@@ -89,6 +89,9 @@ auditLog:
   dynamicBackend:
     # true here configures an AuditSink who will receive the K8s audit logs
     enabled: false
+    # define if auditsink client config should point to a fixed url, not the
+    # default webserver service
+    url: ""
```

Comment:
How can folks find the URL to put in here?

Reply:
cc'ing @nibalizer @maxgio92 
WDYT?

Reply:
I think that since the only part of the endpoint that would change would be the `scheme`, it could be specified whether to enable `plain` HTTP communication to the Falco Audit Backend, in which case:
- the `url` could be made up by the Falco service DNS name as so:
`http://<falco service name>.<falco namespace>:<falco listen port>/<falco audit path>`
- otherwise the `service` reference.

or did I miss something?

Reply:
AFAIK, the DNS cannot be used (see [here](https://kubernetes.io/docs/tasks/debug-application-cluster/audit/#url)) thus it cannot be automatically filled in by helm.

Reply:
I was clearly missing something :)

Reply:
Ok, I have added a note in the docs :point_right: https://github.com/falcosecurity/charts/pull/33/commits/02cdb50513212bb375c2e198ac3febd8bc58b914